src/frontend/src/pages/TeamsPage/ProjectCardsView.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, CardContent, Typography } from '@mui/material';
+import PageBlock from '../../layouts/PageBlock';
+import { Link as RouterLink } from 'react-router-dom';
+import { Link } from '@mui/material';
+import { ProjectPreview } from 'shared';
+import { routes } from '../../utils/routes';
+import { fullNamePipe, wbsPipe } from '../../utils/pipes';
+import { useSingleProject } from '../../hooks/projects.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { Grid } from '@mui/material';
```

Comment:
I tend to be a stickler for the details, so take these with a grain of salt, but: disorganized imports bug me.

1. You've got multiple different separate lines importing from `'@mui/material';`, they should all be combined into one import statement
2. You're mixing package imports, local imports, component imports, etc and it's all over the place. I find it to be much more readable code if you first list all package imports (all from the same package should be together), all other local imports together, and then all component imports together.